packages/core/src/sdk/search/state.ts

```diff
@@ -6,10 +6,15 @@ export const useApplySearchState = () => {
 
   return useCallback(
     (url: URL) => {
+      const currentUrl = new URL(window.location.href)
```

Comment:
question: I'm wondering if it's necessary since this hook is called on `onChange`. 
Removing the condition related to this `currentUrl` doesn't affect the fix you made.
```suggestion
```

Reply:
removed as well :)